android/src/main/java/com/iterable/reactnative/Serialization.java

```diff
@@ -0,0 +1,34 @@
+package com.iterable.reactnative;
+
+import androidx.annotation.Nullable;
+
+import com.iterable.iterableapi.IterableInAppCloseAction;
+import com.iterable.iterableapi.IterableInAppDeleteActionType;
+import com.iterable.iterableapi.IterableInAppLocation;
+
+public class Serialization {
+
+    static IterableInAppLocation getIterableInAppLocationFromInteger(@Nullable Integer location) {
+        if (location == null || location >= IterableInAppLocation.values().length || location < 0) {
+            return null;
+        } else {
+            return IterableInAppLocation.values()[location];
+        }
+    }
+
+    static IterableInAppCloseAction getIterableInAppCloseSourceFromInteger(@Nullable Integer location) {
```

Comment:
Where is `IterableInAppCloseAction` class defined? I don't see this in the list of files.

Reply:
Its an enum defined in our IterableApi SDK like this
<img width="435" alt="image" src="https://user-images.githubusercontent.com/22951141/80688984-81201880-8a81-11ea-9313-6503eff03129.png">

Reply:
Sorry I must be missing something. Which java file is this in?

Reply:
Is it possible that you forgot to check in the file?

Reply:
Ah got it. Sorry.
I thought this was somewhere in RN sdk codebase.